src/ensembl/src/content/app/browser/browser-image/BrowserImage.tsx

```diff
@@ -14,9 +14,10 @@
  * limitations under the License.
  */
 
-import React, { useRef, useEffect, useMemo } from 'react';
+import React, { useRef, useEffect, useMemo, useCallback } from 'react';
 import { useDispatch, useSelector } from 'react-redux';
 import classNames from 'classnames';
+import debounce from 'lodash/debounce';
```

Comment:
I'll just leave this comment here, so we don't forget.

The use of debounce on the chrome side should not be necessary. If the genome browser is currently reporting intended locations while the user is still dragging the canvas or zooming in/out, it suggests that something is broken in the genome browser. The whole purpose of intended location was to notify the browser has reached its destination and the user has completed their interaction with it. This is how it's currently working in production.